### DISCUSSION TITLE
New release on pip please

### DISCUSSION THREAD

**Discussion by felixSpark** [PydPiper/pylightxl#30] [General]
Hi would love to have a new release on pip, please.

This code change fixes exactly a problem I'm having.
https://github.com/PydPiper/pylightxl/commit/e825e54214e71138aad4c54d8e12dd9843d0de1b

Thank you

**Comment by PydPiper** [comment]
Hi Felix,

I pushed the new version out just now! Let me know if there’s anything else
:) and thanks for considering using pylightxl

On Tue, Jan 26, 2021 at 4:06 PM felixSpark <notifications@github.com> wrote:

> Hi would love to have a new release on pip, please.
>
> This code change fixes exactly a problem I'm having.
> e825e54
> <https://github.com/PydPiper/pylightxl/commit/e825e54214e71138aad4c54d8e12dd9843d0de1b>
>
> Thank you
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/PydPiper/pylightxl/discussions/30>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIOU6F22OGN23S75UIKQMJDS35KILANCNFSM4WUKMIHA>
> .
>
-- 
Best,
Viktor Kis
